src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasures.ts

```diff
@@ -92,7 +102,27 @@ export function migrateCalculatedMeasures(
     ([measureName, record]) => {
       const cubeNames = measureToCubeMapping[measureName];
       if (!cubeNames) {
-        // TODO add a warning in the report that the calculated measure was not migrated since it's not used anywhere.
+        // `legacyCalculatedMeasuresFolder.children` contains `content` and `structure` properties.
+        const filesAncestry = _getFilesAncestry(
+          legacyCalculatedMeasuresFolder?.children?.structure!,
+        );
+        const calculatedMeasureId = calculatedMeasureNamesAndIds[measureName];
+        const folderId = filesAncestry[calculatedMeasureId].map(({ id }) => id);
+        const folderName = filesAncestry[calculatedMeasureId].map(
+          ({ name }) => name,
+        );
+        _addErrorToReport(errorReport, {
```

Comment:
Let's incremented the `removed` counter in this case.
In the other cases, let's increment the corresponding counter as well (`failed` or `success`).